src/lua-runtime-fastpm.lua

```diff
@@ -98,7 +98,9 @@ schema.declare{name='read_powerspectrum', type='file', help='file to read the li
 schema.declare{name='read_lineark_ncdm', type='string', help='file to read the lineark of ncdm.'}    --file?
 schema.declare{name='read_powerspectrum_ncdm', type='file', help='file to read the linear power spectrum of ncdm.'} 
 
-schema.declare{name='linear_density_redshift', type='number', default=0, help='redshift of the input linear density field. '}
+schema.declare{name='linear_density_redshift', type='number', default=0, help='redshift of the input linear cdm density field. '}
+schema.declare{name='linear_density_redshift_ncdm', type='number', default=0, help='redshift of the input linear ncdm density field. '}
```

Comment:
these similarly looking names suggest we may want to re-organize the parameters somehow. Any ideas?